### DISCUSSION TITLE
PDF of Cloudscape Design Documentation

### DISCUSSION THREAD

**Discussion by NicoPowers** [cloudscape-design/components#1441] [Q&A] [answered]
Hi,

I would like to use ChatGPT-4 to generate code using the Cloudscape Design, and I was wondering if it would be possible to bundle all the documentation on the cloudscape design website, into a PDF so it can be scrapped by a ChatGPT-4 model.

Thank you!

**Comment by rubencarvalho** [comment, marked as answer]
Hello Nicolas!

We currently don't support bundling and exporting our documentation into a PDF (or any other format) and have no short-term plans to introduce such a feature.

Thank you for reaching out,
Ruben
